vault/logical_system_raft.go

```diff
@@ -506,6 +515,11 @@ func (b *SystemBackend) handleStorageRaftAutopilotConfigUpdate() framework.Opera
 			config.ServerStabilizationTime = time.Duration(serverStabilizationTime.(int)) * time.Second
 			persist = true
 		}
+		disableUpgradeMigration, ok := d.GetOk("disable_upgrade_migration")
```

Comment:
You could use the IsEnterprise const to return an error if someone tries to configure this on OSS.